### PR TITLE
Trying to fix zenodo releases

### PR DESCRIPTION
**Pull request by DanHickstein**
I'm still trying to get the automatic publishing of new releases to Zenodo to work.

For v0.8.0, we got the following response from Zenodo:
```
{
    "errors": "Unknown error occured."
}
```
which is super helpful. This was the "payload" delivered to Zenodo from GitHub [[gist link](https://gist.github.com/DanHickstein/e6a840e1e2d2adccda8811cf467a77d0)]

I then realized that many of the author names on Zenodo for v0.7.6 were just the github user names, and not our real names. I figured out that we can make a `.zenodo.json` file in out repository that contains our corrected names.

Then, I released v0.8.1, and got the following error:
```
{
    "errors": "Metadata file \".zenodo.json\" is not valid JSON."
}
```
Which is legit, because I had some extra commas in the .json file. I removed these, and used https://jsonlint.com/ to ensure that the .json file was valid.

So, I guess we get to try again when we release version 0.8.2, whenever we do that.

EDIT: I also includes some information in CONTRIBUTING.rst saying that people can include their names in that file to be listed as authors on Zenodo.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
-      "name": "Yurchak, Roman",
+      "name": "Yurchak, Roman"
     },
     {
-      "name": "Ryazanov, Mikhail",
+      "name": "Ryazanov, Mikhail"
     },
     {
-      "name": "Shih, Gilbert",
+      "name": "Shih, Gilbert"
```
[the remark this answers is not in the document]
Comment (DanHickstein) [reply .zenodo.json]: Whoops! Missed DJ! My bad!!

Shih is ldes89150, hasn't been involved for a while, but made some significant contributions in the past. You can see the current contributors list here: https://github.com/PyAbel/PyAbel/graphs/contributors

**Comment by MikhailRyazanov** [comment]
Are we waiting for @rth's approval?

If this can be merged, I'm going to open a PR for the [forward BASEX transform](https://github.com/MikhailRyazanov/PyAbel/tree/forward) and then for the [documentation updates](https://github.com/MikhailRyazanov/PyAbel/tree/doc).

**Comment by rth** [comment]
> Are we waiting for @rth's approval?

Please don't -- I think 3 reviews are quite enough :)

**Comment by MikhailRyazanov** [comment]
:–) OK, thanks @rth!

@DanHickstein, can this be merged now?

**Comment by DanHickstein** [comment]
Yup!

**Pull request merged**